src/DynamoCore/Library/MemberDocumentNode.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Dynamo.DSEngine
+{
+    class MemberDocumentNode
+    {
+        private readonly string fullyQualifiedName;
+        private string summary;
+        private string searchTags;
+        private readonly Dictionary<string, string> parameters;
+
+        public string FullyQualifiedName { get { return fullyQualifiedName; } }
+
+        public string Summary
+        {
+            get
+            {
+                if (String.IsNullOrEmpty(summary))
+                    return String.Empty;
```

Comment:
Hi @aosyatnik, as I mentioned in the previous comment, this `String.IsNullOrEmpty` is not required, because we made sure no one can set `summary` to `null`. Same for `searchTags` property below.